src/main/scala/org/ergoplatform/nodeView/wallet/ErgoWalletActor.scala

```diff
@@ -597,7 +597,7 @@ class ErgoWalletActor(settings: ErgoSettings, boxSelector: BoxSelector)
     if (secrets.size == 1) {
       Success(DerivationPath(List(0, 1), publicBranch = false))
     } else {
-      nextPath(List.empty, secrets.map(_.path.decodedPath.tail))
+      nextPath(List.empty, secrets.map(_.path.decodedPath.tail.toList))
```

Comment:
Why this is changed?

Reply:
@aslesarenko DerivationPath.decodedPath in the wallet was changed from List to Seq. Actually, I've updated this code in https://github.com/ergoplatform/ergo/pull/852/commits/d9b99f333ae6980244010cb842b7581adb0fe44c
with a better fit for this change.